printer_zpl2/models/printing_label_zpl2.py

```diff
@@ -504,7 +504,9 @@ def _generate_labelary_image(self):
                 zpl_file = self._generate_zpl2_data(record, labelary_emul=True, **extra)
                 files = {"file": zpl_file}
                 headers = {"Accept": "image/png"}
-                response = requests.post(url, headers=headers, files=files, stream=True)
+                response = requests.post(
+                    url, headers=headers, files=files, stream=True, timeout=30
```

Comment:
This change should normally be outside of the migration commit as it's not tied to the Odoo migration at all, and could benefit to other Odoo versions.

Not blocking.